x-pack/plugins/stack_connectors/server/connector_types/webhook/index.test.ts

```diff
@@ -339,12 +341,42 @@ describe('execute()', () => {
       params: { body: 'some data' },
       configurationUtilities,
       logger: mockedLogger,
+      connectorMetricsCollector,
     });
 
     delete requestMock.mock.calls[0][0].configurationUtilities;
     expect(requestMock.mock.calls[0][0]).toMatchInlineSnapshot(`
       Object {
         "axios": undefined,
+        "connectorMetricsCollector": ConnectorMetricsCollector {
+          "logger": Object {
+            "context": Array [],
+            "debug": [MockFunction] {
+              "calls": Array [
+                Array [
+                  "response from webhook action \\"some-id\\": [HTTP 200] ",
+                ],
+              ],
+              "results": Array [
+                Object {
+                  "type": "return",
+                  "value": undefined,
+                },
+              ],
+            },
+            "error": [MockFunction],
+            "fatal": [MockFunction],
+            "get": [MockFunction],
+            "info": [MockFunction],
+            "isLevelEnabled": [MockFunction],
+            "log": [MockFunction],
+            "trace": [MockFunction],
+            "warn": [MockFunction],
+          },
+          "metrics": Object {
```

Comment:
Since we added a logger to the request call, i had to add the logger to the existing snapshot as well.
There is a second one down there, on line 385.

Reply:
I don't think we should be checking the logger like this.  Check for expected calls to the logger in the usual way, if we need to, but don't include it in a snapshot.  Either delete the property before doing the match snapshot, or create some new object with just the things to check.  

What I'm afraid of is this breaking when we add debug logs (or any other logs) in  some future PR.  Might as well  prevent that from breaking now.  :-)

Reply:
i switched to `toMatchSnapshot` and removed logger here but there are a lot of tests like this in the code.

But these are not testing the logger usage, these are just the logger object we passed as param. so Sdding or removing logging method would not break the test. 

Someone should modify the logger on Kibana level to get this break.